aeon/dj_pipeline/analysis/visit_analysis.py

```diff
@@ -388,6 +388,86 @@ def make(self, key):
             self.Nest.insert(in_nest_times)
             self.FoodPatch.insert(in_food_patch_times)
 
+    @classmethod
+    def _get_foraging_bouts(
+        cls,
+        visit_per_day_row,
+        wheel_dist_crit=None,
+        min_bout_duration=None,
+        using_aeon_io=False,
+    ):
```

Comment:
Please add a short docstring to this function, explaining what it is used for.
Also, what is the `visit_per_day_row` expected to be? A pandas series/dataframe? I think that would put too much burden on the users to be able to use this method, maybe take some sort of a `visit_day_key`?

Reply:
Where is this class method expected to be used? If it is expected to be used by another class/function elsewhere, then this should not be a hidden `_get_foraging_bouts` method - remove the leading `_` in the method name/

Reply:
Yeah, good point. I just designed this way to use the `df.apply` method, but it makes things harder to read or debug. Making it a standalone helper function should be better